_scripts/instruction-widget/install.js

```diff
@@ -84,16 +85,24 @@ module.exports = function(context) {
   debian_install = function() {
     template = "debian";
 
+    if (context.distro == "ubuntu") {
+      context.above_4 = false;
+    }
     // Debian Jessie backports.
     if (context.distro == "debian" && context.version == 8) {
       context.backports_flag = "-t jessie-backports";
     }
 
     // Set package based on webserver.
-    // TODO: I don't think these packages are correct
     if (context.webserver == "apache") {
       context.package = "python-letsencrypt-apache";
     }
+    if (context.id == "debiantesting") {
+      if (context.webserver == "apache") {
+        context.package = "python-certbot-apache";
+      }
+      context.package = "python-certbot"
```

Comment:
This will override setting package to `python-certbot-apache` in the body of the `if` statement above.